Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all image test
+
+IMAGE_NAME=codeclimate/codeclimate-duplication
+
+all: image test
+
+image:
+	docker build -t $(IMAGE_NAME) .
```

Comment:
Not to my knowledge. I'm not 100% clear on the details, but I know if I `docker build --rm ... && docker build --rm` the second is still all Cached steps.